src/gui/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@ set(QTLIBS Qt5::Network Qt5::Widgets)
 include_directories(..)
 qt5_add_resources(NEOVIM_RCC_SOURCES data.qrc)
 if(WIN32)
+	if (${USE_STATIC_QT})
+		add_definitions(-DUSE_STATIC_QT)
+		link_directories($ENV{QTDIR}/share/qt5/plugins/platforms)
```

Comment:
Is this link_directories necessary?

If it is, I think you can use ${CMAKE_PREFIX_PATH} instead of $ENV{QTDIR}

Reply:
Sure need, libqwindows.a store in that path.
Done